src/components/Browse/sortByOption.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { updateBrowseSort } from '../../redux/actions'
+import PropTypes from 'prop-types'
+import { sortByOptionsFormattedMap } from '../../assets/constants'
+
+export default function SortByOption ({ sortBy }) {
+  const dispatch = useDispatch()
+
+  const sortByState = useSelector(state => state.browse.filters.sortBy)
+
+  const clickHandler = () => {
+    dispatch(updateBrowseSort(sortBy))
+  }
+
+  const sortByOptionClassName = sortByState === sortBy
```

Comment:
You can move this directly into line 21 (I think it's better that way but kinda a personal preference thing)